packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -538,6 +538,25 @@ const ruleset: any = {
       },
     },
 
+    ///
+    /// ARM rules for operations API
+    ///
+
+    OperationsApiSchemaUsesCommonTypes: {
+      description: "Operations API path must follow the schema provided in the common types.",
+      message: "{{description}}",
+      severity: "error",
+      resolved: false,
+      formats: [oas2],
+      given: "$[paths,'x-ms-paths'][?(@property.match(/\\/providers\\/\\w+\\.\\w+\\/operations$/i))].get.responses.200.schema.$ref",
```

Comment:
why are we checking only for get?
Will we not have a Put/Patch/Delete for Operations API?

Reply:
Operations API only needs the Get endpoint. Only 200 because that is the response code that would return the common type we are checking for in this rule